app/model/ProgressStatuses.scala

```diff
@@ -121,6 +121,7 @@ object ProgressStatuses {
   case object ASSESSMENT_CENTRE_AWAITING_RE_EVALUATION extends ProgressStatus(ApplicationStatus.ASSESSMENT_CENTRE)
   case object ASSESSMENT_CENTRE_PASSED extends ProgressStatus(ApplicationStatus.ASSESSMENT_CENTRE)
   case object ASSESSMENT_CENTRE_FAILED extends ProgressStatus(ApplicationStatus.ASSESSMENT_CENTRE)
+  case object ASSESSMENT_CENTRE_FAILED_NOTIFIED extends ProgressStatus(ApplicationStatus.ASSESSMENT_CENTRE)
```

Comment:
Thanks for adding this for me.

I know don't look a gift-horse in the mouth, but maybe you can also add `FSB_FAILED_NOTIFIED` 😳

Reply:
Go on then :roll_eyes: